features/errors/internal_server_error.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Page not found error page
```

Comment:
Should the feature have a different name to the not found one?

Reply:
Ah yes, I appear to have got those slightly muddled!